src/plugin.jl

```diff
@@ -346,6 +346,18 @@ you should implement this function and return `true`.
 """
 needs_username(::Plugin) = false
 
+"""
+    pkg_name_from_pkg_dir(pkg_dir::AbstractString)
+
+Return package name of package at `pkg_dir`, i.e., `basename(pkg_dir)` excluding any
+`.jl` suffix, if present. For example, `foo/bar/Whee.jl` and `foo/bar/Whee` both
+return `Whee`.
+"""
+function pkg_name_from_pkg_dir(pkg_dir::AbstractString)
+    pkg = basename(pkg_dir)
+    return endswith(pkg, ".jl") ? pkg[1:end-3] : pkg
```

Comment:
we could use `chopsuffix` (but idk what Julia version that was introduced in, so maybe we can't)

```suggestion
    return chopsuffix(pkg, ".jl")
```

Reply:
TIL! Looks like it shows up in 1.8...